uui/components/buttons/Button.tsx

```diff
@@ -18,7 +20,7 @@ export interface ButtonMods {
 }
 
 export type ButtonProps<PropType = ButtonMods> =
```

Comment:
Why do we need a generic here? Can we just omit 'count' prop and add ButtonMods?
export type ButtonProps = Omit<uuiButtonProps, 'count' | 'indicator'> & ButtonMods

Reply:
fixed it